controllers/cinderscheduler_controller.go

```diff
@@ -87,7 +79,7 @@ type CinderSchedulerReconciler struct {
 
 // Reconcile -
 func (r *CinderSchedulerReconciler) Reconcile(ctx context.Context, req ctrl.Request) (result ctrl.Result, _err error) {
-	_ = log.FromContext(ctx)
+	log := GetLog(ctx, "CinderBackup")
```

Comment:
-1: All calls in this file should be using `CinderScheduler` or `CinderSch` whatever we prefer.

Reply:
right , correcting